bridge/cypress/fixtures/sequences.sockshop.json

```diff
@@ -480,29 +498,6 @@
           "latestEvaluation": {
             "result": "pass",
             "score": 0
-          },
-          "latestEvent": {
-            "type": "sh.keptn.event.dev.delivery.finished",
-            "id": "0838ff2c-f736-41f1-9d7e-5e420f07830d",
-            "time": "2021-07-15T12:02:45.561Z"
-          }
-        },
-        {
-          "name": "staging",
-          "image": "docker.io/keptnexamples/carts:0.12.3",
-          "latestEvaluation": {
-            "result": "pass",
-            "score": 0
-          },
-          "latestEvent": {
-            "type": "sh.keptn.event.staging.rollback.finished",
-            "id": "5d16ec19-e0ef-453d-b312-03e5988a0905",
-            "time": "2021-07-15T12:13:40.942Z"
-          },
-          "latestFailedEvent": {
-            "type": "sh.keptn.event.staging.rollback.finished",
-            "id": "5d16ec19-e0ef-453d-b312-03e5988a0905",
-            "time": "2021-07-15T12:13:40.942Z"
```

Comment:
What is the reason why you removed the staging stage and latestEvent of the dev stage here?